lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -658,8 +658,11 @@ namespace Js
         this->EnsureClearDebugDocument();
         if (this->debugContext != nullptr)
         {
-            this->debugContext->GetProbeContainer()->UninstallInlineBreakpointProbe(NULL);
-            this->debugContext->GetProbeContainer()->UninstallDebuggerScriptOptionCallback();
+            if(this->debugContext->GetProbeContainer())
```

Comment:
that's a thowing OOM, you won't get there if OOM happens.  here it's in the cleaning path.